go/grpcweb/options.go

```diff
@@ -92,6 +96,15 @@ func WithWebsockets(enableWebsockets bool) Option {
 	}
 }
 
+// WithWebsocketPingInterval enables websocket keepalive pinging with the configured timeout (in seconds).
```

Comment:
This is a `time.Duration` now so we don't need to specify the unit anymore.
```suggestion
// WithWebsocketPingInterval enables websocket keepalive pinging with the configured timeout.
```

Reply:
十分感谢您一直以来的帮助。昨天，在提交之前我已经在我的环境中测试通过了。
1.在未使用websocket keepalive pinging时，超时后没有Ping消息，并且60s后就断开了连接。
![图片](https://user-images.githubusercontent.com/16277126/65089071-54d1f300-d9ee-11e9-8711-2df66773519e.png)
2.在将websocket_ping_interval设置为20s时，每次超时20s，会发送Ping消息，并且超时时间比较精确，同时，nginx不在断开连接。
![图片](https://user-images.githubusercontent.com/16277126/65090302-d7f34900-d9ee-11e9-8aaf-fde83c7aaf2e.png)
这与预期的结果相吻合。

Thank you very much for your continuous help.Yesterday, I tested it in my environment before submitting it.
1.When the websocket keepalive pinging is not used, there is no Ping message after timeout, and the connection is disconnected after 60s.
2.When the websocket_ping_interval is set to 20s, a Ping message is sent with each 20s timeout, and the timeout time is accurate, and nginx is no longer disconnected.
This is in line with the expected result. （Translations courtesy of google translate）